### PR TITLE
Summer Release All Custom Labels

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
As a user I want to have all fields, help text, and messages in my language so that I can use the product.

# Critical Changes

# Changes
Custom Label:
- healthLabelIntro

# Issues Closed

# Community Ideas Delivered

# New Metadata
New Custom Labels:
- stgRDNamingDescription
- stgRDNamingDisable
- stgRDNamingOptionLabel
- healthLabelDataBatchNotRun
- healthLabelDisableRecordBatch
- healthLabelDisableRecordBatchDescription
- RD2_EntryFormCustomFieldsSectionHeader
- RD2_PauseClosedRDErrorMessage
- RD2_PauseDescription
- RD2_PauseHeader
- RD2_PauseFirstDonationDateDynamicText

# Deleted Metadata

### CODE REVIEW

**Comment by andrewyu-salesforce** [comment]
**lurch: attach W-042001

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007hbOOQAY" target="_blank">W-042001</a>

**Comment by andrewyu-salesforce** [comment]
@force2b @sam-knox The new labels are ready for reviewed.

**Comment by andrewyu-salesforce** [comment]
@sam-knox Thanks for the review. The label is updated

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -7403,6 +7443,30 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>healthLabelContactData</shortDescription>
         <value>Contact Data</value>
     </labels>
+    <labels>
+        <fullName>healthLabelDataBatchNotRun</fullName>
+        <categories>Health Check</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```
Comment (force2b) [inline src/labels/CustomLabels.labels:7450]: Protected needs to be true. There are two others that need fixing as well.

Comment (andrewyu-salesforce) [reply src/labels/CustomLabels.labels]: @force2b Thanks for the review. The code is updated and is ready for review again.

**Comment by rponti-sforg** [comment]
QE testing is complete for this PR and associated WI.  Based on the WI Description, ACs, 'child' WIs stated in the ACs, and Test Charter feedback, tests have now passed.

For further information, please see:
[W-042001](https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000007hbOOQAY/view)
[Test Charter](https://salesforce.quip.com/QuIQAeM3sXE7#bQaACA1fPyM)

Will set WI status to '**Closed**' once the post merge builds have passed / success.

**Pull request merged**